packages/dashboard-core-plugins/src/panels/NotebookPanel.tsx

```diff
@@ -578,6 +646,48 @@ class NotebookPanel extends Component<NotebookPanelProps, NotebookPanelState> {
     }
   }
 
+  handleMinimap() {
```

Comment:
In terms of naming of these functions, I'd make the following changes:
`handleMinimap` => `updateEditorMinimap`: Makes it clear that the function is updating the child editor minimap settings
`handleMinimapWrapper` => `handleMinimapChange`: Makes it clear that the function is handling the option being changed
`handleWordWrap` => `handleWordWrapChange`: Same as above. Also, instead of calling `this.editor.updateOptions` right in here, I'd have an `updateEditorWordWrap` similar to `updateEditorMinimap`, and call that from `componentDidUpdate` when the minimap settings change in the state. I think that will clean it up a bit.